build.sh

```diff
@@ -200,7 +200,12 @@ ruby-*)
   if [[ $RUBY = *head* ]]; then
     EXTRA_FLAGS="--rubygems ignore"
   fi
-  announce rvm install $RUBY $EXTRA_FLAGS --verify-downloads 1 $MOVABLE_FLAG --disable-install-doc -C --without-tcl,--without-tk,--without-gmp;;
+  CONFIGURE_OPTS=( --without-tcl --without-tk --without-gmp )
+  case $RUBY in
+  ruby-2.4*|ruby-2.5*)
```

Comment:
Assuming that the flag works for all future versions, the logic should be inverted. Do not add to `CONFIGURE_OPTS` if `$RUBY` is matches `ruby-2.[0-3]*`. Otherwise (`ruby-2.4*` and up, as well as `ruby-head`), add `--enable-install-static-library`.